src/static/css/cards.css

```diff
@@ -0,0 +1,37 @@
+.card-image{
+  display:block;
+  max-width:35% !important;
```

Comment:
@denesnori see here https://developer.mozilla.org/en/docs/Web/CSS/Specificity.

Might help. I know the your pain, the idea is to try and make the selectors more specific